component/otelcol/processor/tail_sampling/types.go

```diff
@@ -86,18 +82,19 @@ type NumericAttributeConfig struct {
 	MinValue int64 `river:"min_value,attr"`
 	// MaxValue is the maximum value of the attribute to be considered a match.
 	MaxValue int64 `river:"max_value,attr"`
+	// InvertMatch indicates that values must not match against attribute values.
+	// If InvertMatch is true and Values is equal to '123', all other values will be sampled except '123'.
+	// Also, if the specified Key does not match any resource or span attributes, data will be sampled.
+	InvertMatch bool `river:"invert_match,attr,optional"`
```

Comment:
There are 2 new config options from otel I wired in as part of this PR. You will find them both documented in the component docs.